### PR TITLE
Update Italian

### PR DESCRIPTION
**Pull request by Raffaele74**
Add and translate line 196

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -227,8 +228,32 @@
...
+    <string name="yellow_theme_mode" translatable="false">Yellow</string>
+    <string name="amber_theme_mode" translatable="false">Amber</string>
+    <string name="orange_theme_mode" translatable="false">Orange</string>
+    <string name="deep_orange_theme_mode" translatable="false">Deep Orange</string>
     <string name="theme_title">Scegli un tema</string>
     <string name="theme_summary">Scegli il tuo tema predefinito</string>
+    <string name="theme_color_summary">Scegli il colore di evidenza</string>
+    <string name="theme_color_title">Colore evidenziatore</string>
+    <string name="logo_designer" formatted="true" translatable="false">Logo Designer (%s)</string>
+    <string name="google_plus" translatable="false">Google+</string>
```
Comment (yakov116) [inline app/src/main/res/values-it/strings.xml]: translatable="false"
Can you please remove all translatable resources from the file

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -229,6 +230,9 @@
     <string name="mark_notification_summary">Premendo su una notifica non la segnerà come letta.</string>
     <string name="theme_title">Scegli un tema</string>
     <string name="theme_summary">Scegli il tuo tema predefinito</string>
+    <string name="theme_color_summary">Scegli il colore di evidenza</string>
+    <string name="theme_color_title">Colore evidenziatore</string>
+    <string name="website">Website</string>
```
Comment (yakov116) [inline app/src/main/res/values-it/strings.xml]: This one was lucky to make it in 2 times :wink:

File: app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (yakov116) [inline app/src/main/res/values-it/strings.xml]: This is done but the website sting is there twice please remove one of them

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by yakov116** [comment]
@Raffaele74 do you have an XDA profile that I can give you a mention?

**Comment by Raffaele74** [comment]
No, I haven't.Thankyou anyway!Cheers, Raffaele



----Messaggio originale----

Da: notifications@github.com

Data: 29-mag-2017 17.50

A: "k0shk0sh/FastHub"<FastHub@noreply.github.com>

Cc: "Raffaele74"<raffaele.imperio@tin.it>, "Mention"<mention@noreply.github.com>

Ogg: Re: [k0shk0sh/FastHub] Update Italian (#566)




@Raffaele74 do you have an XDA profile that I can give you a mention?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/k0shk0sh/FastHub","title":"k0shk0sh/FastHub","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/k0shk0sh/FastHub"}},"updates":{"snippets":[{"icon":"PERSON","message":"@yakov116 in #566: @Raffaele74 do you have an XDA profile that I can give you a mention?"}],"action":{"name":"View Pull Request","url":"https://github.com/k0shk0sh/FastHub/pull/566#issuecomment-304690899"}}}

**Pull request merged**